### DISCUSSION TITLE
How to get right dimensions in Xbox?

### DISCUSSION THREAD

**Discussion by gabrigode** [microsoft/react-native-windows#10398] [Q&A] [answered]
Evertime I try to load my RNW app on Xbox with a 4K display the dimensions are all wrong. It loads with 960 x 540 size.

I've been using useWindowDimensions hook to get the dimensions:

`const { height, width } = useWindowDimensions();`

**Comment by chrisglein** [comment, marked as answer]
You have the right method, but it doesn't seem to be giving you the results you'd expect. Sounds like an issue. I'll create one from this question.

https://github.com/microsoft/react-native-windows/issues/10400
